app/services/cognito/cognito_base_service.rb

```diff
@@ -29,7 +29,12 @@ def forgot_password_error_path
 
     # The user pool ID for the user pool where we want to update user attributes
     def user_pool_id
-      ENV['AWS_COGNITO_USER_POOL_ID']
+      if ENV['AWS_COGNITO_USER_POOL_ID'].start_with?('arn:') 
+        # This assumes a fixed cognito pool ID length of 19 chars
+        test = ENV['AWS_COGNITO_USER_POOL_ID'].reverse[0...19].reverse
+      else
+        ENV['AWS_COGNITO_USER_POOL_ID']
+      end
```

Comment:
This can be further simplified to "always take last 19 characters" as it will behave the same way for ARN and non-arn user pool IDs :)
```suggestion
      ENV['AWS_COGNITO_USER_POOL_ID'][-19,19]
```